### PR TITLE
Updating Docs via Automated Script

### PR DESCRIPTION
**Pull request by keithing**

### CODE REVIEW

**Comment by keithing** [comment]
Let's wait for #42, then we'll rebase this.

**Comment by keithing** [comment]
@patr1ckm this is ready for review! It's built off v1.0.0.

File: articles/civis_ml.Rmd [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+
+[CivisML](https://www.civisanalytics.com/blog/civisml-scikit-learn-at-scale/) is a machine learning service on Civis Platform that makes this as painless as possible. You can fit many different models, do extensive metaparameter tuning, and score data sets with millions of observations stored in remote databases. Once these models are built, they live in Civis Platform permanently and can be included into production pipelines. Results can be easily incorporated into reports and dashboards.
+
+CivisML is built in Python using [scikit-learn](http://scikit-learn.org/stable/), and leverages AWS behind the scenes for efficient distributed computing. However, most of its features can be used through R without knowledge of Python or AWS with the `civis_ml` function in `civis`.
+
+While `civis_ml` is a complex function with many arguments, basic machine learning modeling and scoring can be easily carried out easily. We illustrate several features of `civis_ml` with data from a fictious company called Brandable, who is looking to predict which customers are likely to upgrade from the free to the premium service.
```
Comment (patr1ckm) [inline articles/civis_ml.Rmd]: can be easily carried out easily -> can be easily carried out

Comment (patr1ckm) [reply articles/civis_ml.Rmd]: my fault, sorry

**Comment by keithing** [comment]
okay, this was rebased after #44

**Comment by patr1ckm** [comment]
LGTM

**Pull request merged**